src/main/java/com/searchcode/app/App.java

```diff
@@ -515,6 +516,10 @@ public static void main( String[] args ) {
                 return new ApiResponse(false, "repobranch is a required parameter");
             }
 
+            if (repomasks == null) {
```

Comment:
I think we need to create a new endpoint for this api call such as /api/repo/add/v2/ to avoid breaking compatibility with existing clients